templates/quickstart-database-for-atlassian-services.yaml

```diff
@@ -127,6 +127,10 @@ Parameters:
       can include numbers, lowercase letters, uppercase letters, and hyphens (-).
       It cannot start or end with a hyphen (-).
     Type: String
+  QSS3BucketRegion:
+    Default: ''
+    Type: String
+    Description: The AWS Region where the Quick Start S3 bucket is hosted.
```

Comment:
Because of the need to maintain backward compatibility we cannot use a default or a nice drop down of values.

Reply:
Correct me if I'm wrong, but is this parameter only used if the customer is using their own S3 bucket? Looking at line 191 it looks like we only use this parameter if the `UsingDefaultBucket` condition is false, i.e. they're using their own S3 bucket.

If that is the case should we put a note that "this parameter is only required if you are using your own S3 bucket for the quickstart templates.

Reply:
Correct! And yes you are right about the description I will update